app.py

```diff
@@ -11,14 +11,14 @@
 # Setup
 #
 
-load_dotenv()
+load_dotenv()         # Load environment variables
```

Comment:
```suggestion
load_dotenv()  # Load environment variables
```
segundo PEP8, comentários vem 2 espaços após o código